etc/org.opencastproject.caption.converters.WebVttCaptionConverter.cfg

```
@@ -1,5 +1,5 @@
 
 # defines the mediapackage element type of the subtitle files
 # possible values: Attachment, Track
-# default: Attachment
-#mediapackage-element-type=Attachment
+# default: Track
+mediapackage-element-type=Track
```

Default values should be commented out.
```suggestion
#mediapackage-element-type=Track
```

---

Woops, how did that happen. sorry and thanks :D